dts/bindings/radio_coex/generic-radio-coex-one-wire.yaml

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2021 Nordic Semiconductor ASA
```

Comment:
commit msg nit: We're not only adding the configuration for 1-wire coex, but it seems like we're adding the whole feature in this commit: dts bindings, Kconfig options and init code. Could you also squash/fixup the documentation commit into this one ?